pkg/imagefilter/export_test.go

```diff
@@ -0,0 +1,3 @@
+package imagefilter
+
+var SupportedFormatterMap = supportedFormatters
```

Comment:
`SupportedFormatters` (the type is already a `map`) is the only bit.

Reply:
grr - overlooked that one, thanks